tests/adapter/dbt/tests/adapter/column_types/test_column_types.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from dbt.tests.util import run_dbt
+from dbt.tests.adapter.column_types.fixtures import macro_test_is_type_sql, model_sql, schema_yml
+
+
+class BaseColumnTypes:
+    @pytest.fixture(scope="class")
+    def macros(self):
+        return {"test_is_type.sql": macro_test_is_type_sql}
+
+    def run_and_test(self):
+        results = run_dbt(["run"])
+        assert len(results) == 1
+        results = run_dbt(["test"])
+        assert len(results) == 1
+
+
+class TestPosgresColumnTypes(BaseColumnTypes):
```

Comment:
Pos***t***gres

And should we consider moving the Postgres specific items into their own directory in order to start treating Postgres as its own independent adapter?

Reply:
This is a good point we are quickly moving towards that moment when we pull it out, but i'm not sure if we are there yet or even if I do need to refer to this as the "Postgres" test yet but did it out of habit. likely could just treat this as the generic test case without mentioning it for now till we do get ready. but do like the idea of starting talks of do we want to go ahead and group those tests now.